src/applications/mhv/secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -158,11 +161,13 @@ const ComposeForm = props => {
           sendData.append('message', JSON.stringify(messageData));
           attachments.map(upload => sendData.append('uploads[]', upload));
           dispatch(sendMessage(sendData, true))
-            .then(() => history.push(Paths.INBOX))
+            .then(() =>
+              navigateToFolderByFolderId(currentFolder?.folderId || 0, history),
+            )
             .catch(setSendMessageFlag(false));
         } else {
           dispatch(sendMessage(JSON.stringify(messageData), false)).then(() =>
-            history.push(Paths.INBOX),
+            navigateToFolderByFolderId(currentFolder?.folderId || 0, history),
```

Comment:
not sure why these are duplicated, but I left the same comment on another PR, depending on how you will be resolving these conflicts. let's rather pull that `0` value from constants `DefaultFolders object`